src/components/FilterDeleteModal/FilterDeleteModal.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Button } from '../Button';
+import { Modal } from '../Modal';
+
+export class FilterDeleteModal extends React.Component {
```

Comment:
Ignore this test comment

Reply:
Ahah t'as enfin découvert cette feature du futur